pkg/apis/openebs.io/v1alpha1/cas_volume.go

```diff
@@ -58,10 +58,10 @@ const (
 	NamespaceKey CASKey = "openebs.io/namespace"
 
 	// PersistentVolumeClaimKey is the key to fetch name of PersistentVolumeClaim
-	PersistentVolumeClaimKey CASKey = "openebs.io/pvc"
+	PersistentVolumeClaimKey CASKey = "openebs.io/persistentvolumeclaim"
 
 	// StorageClassKey is the key to fetch name of StorageClass
-	StorageClassKey CASKey = "openebs.io/storage-class"
+	StorageClassKey CASKey = "openebs.io/storageclass"
```

Comment:
Change in StorageClassKey needs to change in provisioner too, i will raise a PR with changes once this got merge

Reply:
yes.. please